third_party/boringssl/src/crypto/mem.c

```diff
@@ -169,17 +173,22 @@ void OPENSSL_free(void *orig_ptr) {
     return;
   }
 
+#if !defined(OPENSSL_SYS_STARBOARD)
```

Comment:
It would be good to understand why you needed `OPENSSL_SYS_STARBOARD` and what didn't work out of the box.

Reply:
Evergreen builds failed thus:
```
Leaks introduced:
  OPENSSL_memory_alloc
    libboringssl.a
      //third_party/boringssl/boringssl/mem
  OPENSSL_memory_free
    libboringssl.a
      //third_party/boringssl/boringssl/mem
  OPENSSL_memory_get_size
    libboringssl.a
      //third_party/boringssl/boringssl/mem
  sdallocx
    libboringssl.a
      //third_party/boringssl/boringssl/mem
```

See the `WEAK_SYMBOL_FUNC` macros excluded above.